docs/guides/notebooks/configuration/groups.rst

```diff
@@ -0,0 +1,129 @@
+############################
+User groups for file sharing
+############################
+
+Create and manage closed user groups in order to share private files in the ``/project`` directory.
+
+.. _user-group-create:
+
+Create a closed group
+=====================
+
+Comanage webpage
+----------------
+
+* In a browser, navigate to `id.lsst.cloud <https://id.lsst.cloud>`_ and log in.
+
+* In the left menu sidebar, click on "Groups" and then "My Groups".
+
+* At right, a list of all the groups associated with your account will appear.
+
+* In the top horizontal menu, click on "Add Group".
+
+* Set the group properties:
+
+  * The group name should be short and must start with ``g_``.
+
+  * Write a short description.
+
+  * Leave the status as the default "Active".
+
+  * Leave the checkbox next to "Open" unselected, to create a closed group.
```

Comment:
Due to the current limitation in group number we probably want to actively dissuade users from self-joining groups.

Reply:
OK good to know -- I think the phrase "Only group owners can manage group membership." does this ok.